apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -62,76 +62,10 @@ type RoutingConfig struct {
-	PerWorkspace *resource.Quantity `json:"perWorkspace,omitempty"`
-}
-
-type ServiceAccountConfig struct {
-	// ServiceAccountName defines a fixed name to be used for all DevWorkspaces. If set, the DevWorkspace
-	// Operator will not generate a separate ServiceAccount for each DevWorkspace, and will instead create
-	// a ServiceAccount with the specified name in each namespace where DevWorkspaces are created. If specified,
-	// the created ServiceAccount will not be removed when DevWorkspaces are deleted and must be cleaned up manually.
-	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
-	// +kubebuilder:validation:MaxLength=63
-	ServiceAccountName string `json:"serviceAccountName,omitempty"`
-	// Disable creation of DevWorkspace ServiceAccounts by the DevWorkspace Operator. If set to true, the serviceAccountName
-	// field must also be set. If ServiceAccount creation is disabled, it is assumed that the specified ServiceAccount already
-	// exists in any namespace where a workspace is created. If a suitable ServiceAccount does not exist, starting DevWorkspaces
-	// will fail.
-	DisableCreation *bool `json:"disableCreation,omitempty"`
-	// List of ServiceAccount tokens that will be mounted into workspace pods as projected volumes.
-	ServiceAccountTokens []ServiceAccountToken `json:"serviceAccountTokens,omitempty"`
-}
-
-type ServiceAccountToken struct {
-	// Identifiable name of the ServiceAccount token.
-	// If multiple ServiceAccount tokens use the same mount path, a generic name will be used
-	// for the projected volume instead.
-	// +kubebuilder:validation:Required
-	Name string `json:"name"`
-	// Path within the workspace container at which the token should be mounted.  Must
-	// not contain ':'.
-	// +kubebuilder:validation:Required
-	MountPath string `json:"mountPath"`
-	// Path is the path relative to the mount point of the file to project the
-	// token into.
-	// +kubebuilder:validation:Required
-	Path string `json:"path"`
-	// Audience is the intended audience of the token. A recipient of a token
-	// must identify itself with an identifier specified in the audience of the
-	// token, and otherwise should reject the token. The audience defaults to the
-	// identifier of the apiserver.
-	// +kubebuilder:validation:Optional
-	Audience string `json:"audience,omitempty"`
-	// ExpirationSeconds is the requested duration of validity of the service
-	// account token. As the token approaches expiration, the kubelet volume
-	// plugin will proactively rotate the service account token. The kubelet will
-	// start trying to rotate the token if the token is older than 80 percent of
-	// its time to live or if the token is older than 24 hours. Defaults to 1 hour
-	// and must be at least 10 minutes.
-	// +kubebuilder:validation:Minimum=600
-	// +kubebuilder:default:=3600
-	// +kubebuilder:validation:Optional
-	ExpirationSeconds int64 `json:"expirationSeconds,omitempty"`
-}
-
```

Comment:
Note: I reorganized this file to put the `WorkspaceConfig` struct first; these fields are unchanged, just moved.